### PR TITLE
added new comment

### PR DESCRIPTION
**Pull request by VeraDoa**
please check

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# training-pr
\ No newline at end of file
+# training-pr
+hello Olga from Vera
```
Comment (vmolash) [inline README.md]: call me Olya

**Review by vmolash — CHANGES_REQUESTED** [review]
line #3 error in name

File: README.md (hunk 1 above)
Comment (Veradbbs) [reply README.md]: @ Are you sure?

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
-# training-pr
\ No newline at end of file
+# training-pr
```
Comment (vmolash) [inline README.md:1]: comment for line 2

**Comment by VeraDoa** [comment]
@vmolash conflict has resolved, please check

File: README.md (hunk 2 above)
Comment (VeraDoa) [reply README.md:1]: @vmolash added comment for line 2

**Review by vmolash — CHANGES_REQUESTED** [review]
resolve conflict, please

**Review by vmolash — APPROVED** [review]
works

**Pull request merged**